features/adb-openshift.feature

```diff
@@ -18,17 +18,13 @@ Feature: Command output from various OpenShift related commands in ADB
       config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
       config.vm.network :private_network, ip: '<ip>'
       config.vm.synced_folder '.', '/vagrant', disabled: true
-      config.servicemanager.services = 'docker, openshift'
 
-      config.servicemanager.openshift_image_tag = 'v1.2.1'
-      config.servicemanager.openshift_image_name = 'openshift/origin'
-      config.servicemanager.openshift_docker_registry = 'docker.io'
```

Comment:
@LalatenduMohanty It was surplus. 
Previously we had issue in VSM https://github.com/projectatomic/vagrant-service-manager/issues/357 which was unable to handle latest or alpha/beta tags. This issue has been fixed and now these lines are not needed to pin down openshift version